.env

```diff
@@ -19,3 +19,4 @@ REACT_APP_VERSION=$npm_package_version
 REACT_APP_NAME=$npm_package_name
 
 REACT_APP_NODEOS_DEFAULT_PATH=http://localhost:8888
+REACT_APP_API_NETWORK_NAME= 'airwire'
```

Comment:
@tetogomez   @kuronosec 

This environmental variable should only be set for the airwire testnet and airwire mainnet explorers